content/docs/config.toml

```diff
@@ -0,0 +1,34 @@
+baseURL = "http://fshschool.org/"
```

Comment:
Is this whole file a mistake?  It looks like the old "Learn"-based `config.toml` (except with the `theme` set to `"docsy"` instead -- and it's in the wrong place...